mkdocs.yml

```diff
@@ -30,89 +29,96 @@ theme:
 
 nav:
   - Learn:
-    - Welcome: learn/welcome.md
-    - Mainnet: learn/mainnet.md
-    - Features: learn/features.md
-    - Use cases: learn/use-cases.md
-    - Glossary: learn/glossary.md
-    - Networks: learn/networks.md
-    - Blog: learn/blog.md
-    - Explore:
-      - Explorers: explore/explorers.md
-      - Projects: explore/projects.md
-      - Sample apps: explore/sample-apps.md
-    - Advanced:
-      - Tech overview: learn/advanced/overview.md
-      - Data availability: learn/advanced/data-availability.md
+      - Welcome: learn/welcome.md
+      - Mainnet: learn/mainnet.md
```

Comment:
hm I think two spaces rather than four was used intentionally in this section.

Reply:
It got automatically formatted by my editor, the spacing doesn't seem to impact the structure though, unless I missed something?

Reply:
Right, I'd prefer if we try to not change things unless we have reason to! Someone elses editor might have made it the other way.

Reply:
Apparently it detected the spacing indentation to 4 spaces, probably because the indentation was inconsistent in the file. Should be set to 2 spaces everywhere now.

Reply:
thanks!